### DISCUSSION TITLE
Confused by expected value results

### DISCUSSION THREAD

**Discussion by JimCollinsDC** [kyleskom/NBA-Machine-Learning-Sports-Betting#130] [General]
Tonights readout says:
Utah Jazz EV: -7.39
San Antonio Spurs EV: 10.08
To me this looks like the Jazz are expected to lose. ChatGPT agrees with me.
"This appears to be a reference to the expected value (EV) of two basketball teams, the Utah Jazz and the San Antonio Spurs, in a particular context, such as a betting or prediction market.
**The negative number (-7.39) associated with the Utah Jazz EV suggests that they are expected to perform worse than their opponent. The positive number (10.08) associated with the San Antonio Spurs EV suggests that they are expected to perform better than their opponent.**
In general, expected value represents the average outcome of a particular event, taking into account the probability of each possible outcome. In the context of sports betting or prediction markets, expected value is often used to assess the relative strength or weakness of different teams, and to help bettors or investors make informed decisions about where to place their money."

What am I missing here?

**Comment by kyleskom** [comment]
Nope that is all exactly right. It takes probability to win in account with payout, positive EV is what the "best" bet are to make.

**Reply by kyleskom** [reply]
This is also just part of the tools to be using. Account for injures / lineups ect

**Comment by JimCollinsDC** [comment]
So it's saying to take the SPURS?? Ew.
Then why does it also say Utah Jazz (76.1%) vs San Antonio Spurs ?

**Reply by kyleskom** [reply]
Based on payout of odds as well, it thinks its more profitable in the long run to make the bets with positive EV. If a team was expected to win 40% of the time but the payout was say +1000, it makes more sense to take that side of the bet.

**Reply by kyleskom** [reply]
If you were to flip a coin and get 100 bucks if its heads and 1000 bucks if its tails, you'll take tails every single time.

**Reply by nealmick** [reply]
this prediction was correct tho right?   the result was spurs 102 Jazz 94.  odds on my system say -9.5 to jazz.  This game is pretty funny tho spurs were on -16 loss streak.  My model predicted scores 101 - 110.  which did choose under on -9.5 spread but with a very small margin.

https://nbadata.cloud/predict/edit/1428/

**Reply by JimCollinsDC** [reply]
It's not really a prediction of an individual game though. As Kyle pointed out, it's an estimate of your overall return if you bet on the Spurs 100 times at these odds.

**Reply by nealmick** [reply]
I think I understand you.  My only thought is the overall metric of if a prediction is correct or not, only has to do with that one game.  Even if the indicator is based on 100 games, or really any form of prediction.  This docent work if you had two different predictions indicating opposite results..... however you could calculate each prediction separately.  Im not sure if this game is two predictions with opposite results? I think it's predicting the spurs? and if its predicting both, then idk, thats over my head.

**Reply by 512jay** [reply]
The positive expected value bets are just saying based on the pay-out it is better to bet on this game, it's not predicting a winner.

**Reply by nealmick** [reply]
totally could be wrong, I think the expected value is indicating a team within a game, not a game amongst games.  I dont really bet on games and dont fully understand these terms.  I think the metric of if a prediction is correct has to do with which team was indicated to win and if they actually did win with the odds.  whether its outputting a money value of expected returns, or probabilities, or scores, or a 0 or 1, whatever the model is..... from what ive seen bookmakers usually offer home or visitor, and only one wins.
